### PR TITLE
merge with the default 'switch' branch in PullConsumerDemo

### PR DESCRIPTION
**Pull request by J-Cod3r**
### Describe what this PR does / why we need it
make the code clearer and more readable.

### CODE REVIEW

File: sentinel-demo/sentinel-demo-rocketmq/src/main/java/com/alibaba/csp/sentinel/demo/rocketmq/PullConsumerDemo.java [hunk 1]
```diff
@@ -76,14 +76,11 @@ public static void main(String[] args) throws MQClientException {
                     putMessageQueueOffset(mq, nextOffset);
                     consumer.updateConsumeOffset(mq, nextOffset);
                     switch (pullResult.getPullStatus()) {
-                        case FOUND:
-                            break;
-                        case NO_MATCHED_MSG:
-                            break;
                         case NO_NEW_MSG:
                             break SINGLE_MQ;
+                        case FOUND:
```
Comment (yunfeiyanggzq) [inline sentinel-demo/sentinel-demo-rocketmq/src/main/java/com/alibaba/csp/sentinel/demo/rocketmq/PullConsumerDemo.java:81]: emmm,I think both  the previous code and your code are clear, It’s just that everyone’s code habits are different.

Comment (J-Cod3r) [reply sentinel-demo/sentinel-demo-rocketmq/src/main/java/com/alibaba/csp/sentinel/demo/rocketmq/PullConsumerDemo.java:81]: modification suggestions from intellij idea code inspection, so just a suggestions

Comment (sczyh30) [reply sentinel-demo/sentinel-demo-rocketmq/src/main/java/com/alibaba/csp/sentinel/demo/rocketmq/PullConsumerDemo.java:81]: It's okay :)

**Comment by yunfeiyanggzq** [comment]
LGTM

**Comment by sczyh30** [comment]
Thanks for contributing!

**Pull request merged**